README.mdx

```diff
@@ -71,6 +71,71 @@ Calimero Network's story is one of relentless pursuit of a more private,
 efficient, and user-centric decentralized web, setting a new standard for what
 decentralized platforms can achieve.
 
+## Architecture
+
+```mermaid
+graph TD
+    subgraph Datastore
+        DS1[Context]
+        DS2[General]
+        DS3[Network]
+        DS1 --> DS4[Identity]
+        DS1 --> DS5[State]
+        DS1 --> DS6[Transactions]
+        DS2 --> DS7[Scope]
+        DS2 --> DS8[Fragment]
+        DS3 --> DS9[KAD]
+
+        subgraph impl
+          DI1[Memory]
+          DI2[S3]
+          DI3[Level]
+          DI4[RocksDB]
+        end
+    end
+
+    subgraph Blobstore
+        BS1[FS]
+        BS2[S3]
+        BS3[Applications]
+        BS4[Generic Data]
+        BS3 --> BS1
+        BS4 --> BS2
```

Comment:
the File system & S3 are examples of blobstore implementations

while Applications & Generic Data are examples of what's stored inside the blobstore